.env.empty

```diff
@@ -1,8 +0,0 @@
-DISCORD_TOKEN=
```

Comment:
Do we need to delete this file?

Reply:
We probably shouldn't delete this file since we're using it to as a template for `.env`.

Reply:
I misunderstood the first comment, added it back in.